docs/cli_options.md

```
@@ -13,23 +13,23 @@ Use these options to profile with AIPerf.
-│    ENDPOINT-TYPE --endpoint-type                            The endpoint type to send requests to on the server. [choices: chat, completions, embeddings, rankings, responses]        │
+│    ENDPOINT-TYPE --endpoint-type                            The endpoint type to send requests to on the server. [choices: chat, completions, embeddings, rankings]                   │
 │                                                             [default: chat]                                                                                                           │
 │    STREAMING --streaming                                    An option to enable the use of the streaming API. [default: False]                                                        │
 │    URL --url                                            -u  URL of the endpoint to target for benchmarking. [default: localhost:8000]                                                 │
-│    REQUEST-TIMEOUT-SECONDS --request-timeout-seconds        The timeout in floating-point seconds for each request to the endpoint. [default: 600.0]                                 │
+│    REQUEST-TIMEOUT-SECONDS --request-timeout-seconds        The timeout in floating-point seconds for each request to the endpoint. [default: 600.0]                                  │
 │    API-KEY --api-key                                        The API key to use for the endpoint. If provided, it will be sent with every request as a header: Authorization: Bearer   │
 │                                                             <api_key>.                                                                                                                │
 ╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)